release-team/role-handbooks/bug-triage/README.md

```diff
@@ -26,16 +25,24 @@ There are five relevant periods where your workload changes:
 
 ## How You Do Your Job
 
-As Bug Triage lead, it is not your job to fix, label, sort, or gatekeep issues and PRs.  It is your job to get the SIGs, the issue or PR owners, and the key contributors to do it.  So whenever you find a bug that needs to be "fixed" or kicked out of the release, you go through the following escalation:
+As Bug Triage lead, _it is not your job_ to label, sort or gatekeep issues and PRs.
 
-1. Leaving a comment on the GitHub issue or PR, e.g. "This issue hasn't been updated in 3 months.  Are you still expecting to complete it for 1.11?".  It's helpful here to @ mention individuals or SIG ```-bugs``` or ```-pr-reviews``` aliases, e.g. "@kubernetes/sig-node-bugs" or "@kubernetes/sig-network-pr-reviews".
-2. Sending a message to the SIG channel or mailing list about the problem.  It's helpful here to condense multiple issues into a list, e.g. "Hey, these three issues haven't seen any attention, are they still valid for 1.11?"
-3. Messaging individual owners and reviewers via Slack and/or direct email (GitHub notification emails must have been filtered or missed if you're past step #1).  Individual's email addresses may be harder to find than GitHub ID, but ure usually in the git commit history.  Slack handles are often harder yet to find.  There is no master list mapping human names to GitHub ID, email or Slack ID.  If you can't find contact info, asking in the appropriate SIG's Slack channel will usually get you pointed in the right direction.
-4. Escalating to the Release Team Lead with suggestions on what to do with non-responsive issues.
+_Instead_, you should get the assignees, owners, SIG-leads or key contributors to do it, as it is their _job_ to do so.
 
-In general, you should never decide whether something is in or out of a milestone; the SIG or the Release Team Lead needs to do that.  In practice, you should fix anything simple that saves folks time when the intent is obvious or a milestone decision has been made.  For example, you might add/modify kind and priority labels for a PR to match a correlating issue, or make sure an urgently awaited PR is in the milestone so it will pass CI.  See [the documentation for issue kind labels](https://git.k8s.io/community/contributors/devel/release.md#issue-kind-label)
+In general, you shouldn't decide whether something is in or out of a milestone; either the SIG or the Release Team Lead needs to do that.  
+In practice, you should fix anything simple that saves folks time when the intent is obvious or a milestone decision has been made. 
+For example, you might add/modify kind and priority labels for a PR to match a correlating issue, or make sure an urgently awaited PR is in the milestone so it will pass CI.  See [the documentation for issue kind labels](https://git.k8s.io/community/contributors/devel/release.md#issue-kind-label)
```

Comment:
nit: put `kind` and `priority` in backticks

Reply:
done, but still doing lots of changes in the rest of the doc :) will ping when it's ready